docs-src/docs/dataset-exploration/understand-dataset-content.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Understand what your dataset contains
+---
+
+## Understand which scenes you've annotations for in your dataset
+
+To be able to know which scenes you can provide predictions for you need to understand which scenes you've
+annotations for in your dataset. This can be done by using an endpoint for listing annotations in your dataset. The following code snippet by utilizing an endpoint for
+will do this and save it to a csv file.
+
```

Comment:
Maybe add that they currently can not add scenes to their dataset, if they think it is missing. Say that they should contact us in that case.